notifications.json

```diff
@@ -0,0 +1,478 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "version": "1.0",
+        "title": "Notification API",
+        "description": "Notification API"
+    },
+    "host": "e2effzjxvtgbmuilw-mock.stoplight-proxy.io",
+    "basePath": "/v1",
+    "schemes": [
+        "https"
+    ],
+    "paths": {
+        "/Messages/{id}": {
```

Comment:
I think 'Messages', being broader, is more future proof. There's nothing, technically speaking, that prevents the API being used as a two-way means of communication.